package.json

```diff
@@ -7,8 +7,9 @@
     "dev": "vite",
     "build": "vite build",
     "preview": "vite preview",
-    "format:check": "prettier --check .",
     "format:fix": "prettier --write .",
+    "format:check": "prettier --check .",
+    "test": "echo \"No tests specified yet\" && exit 0",
```

Comment:
We are not writting test so there is no need to add the test script, please remove it